src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -5,6 +5,8 @@ export interface NameRequestIF {
   legalType: EntityTypes
   legalName: string
   nrNumber: string
+  expiry?: string
+  status?: string
```

Comment:
Can we use an enum type for this?

Reply:
We can indeed!

Reply:
Fixed in next commit

Reply:
> This was an intentional time to verify the UTC offset, ie, 7:59 UTC is the day before in Pacific (either PST or PDT, not sure).
> This is to ensure that code doesn't just truncate the time and assume the date is correct.
> 
> As long as this is well tested, I don't really care about the times/dates, but this is an area where we often fall down, so please test thoroughly.

Absolutely, in this scenario we don't do anything in the UI with this date afaik. Just display it. 
So if this offset is intentional and consistent from the backend, the ui will reflect the correct date locally. 
On that note though, since it is a very specific offset catering to PST, there could be oddities in other time zones.. but really this should be left to the back end 😩